pharos-cluster.gemspec

```diff
@@ -32,7 +32,7 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "fugit", "~> 1.1.2"
   spec.add_runtime_dependency "rouge", "~> 3.1"
   spec.add_runtime_dependency "tty-prompt", "~> 0.16"
-  spec.add_runtime_dependency "k8s-client", "~> 0.3.1"
+  spec.add_runtime_dependency "k8s-client", "~> 0.3.2"
```

Comment:
Gemfile.lock needs also update?

Reply:
Good catch, I test this in Docker so the `Gemfile.lock` update only happens within the image build.